Tweak: Перевод симптомов, болезней, вирусов, инфекций и состояний

Ohar opened the pull request:
## Changelog
:cl:
tweak: Перевод симптомов, болезней, вирусов, инфекций и состояний
/:cl:

Вливать после [перевода вампиров](https://github.com/ss220-space/Paradise/pull/622) — используются функции оттуда.

Ohar committed:
еревод фраз торговых автоматов

Ohar committed:
Фразы торговых автоматов: мелкие правки

Ohar committed:
Merge branch 'ss220-space:master220' into master220

Ohar committed:
Merge branch 'ss220-space:master220' into master220

Ohar committed:
Перевод способностей вампира

Ohar committed:
Локализован шведский акцент

Ohar committed:
replacetextEx → replacetextEx_char
See https://github.com/ss220-space/Paradise/pull/647#pullrequestreview-878443663

Ohar committed:
swedish desc: change text

Ohar committed:
Round Statistics and Score translation

Ohar committed:
Polly the Parrot: additional phrases

Ohar committed:
Fix spaces → tabs

Ohar committed:
Правка опечатки

Ohar committed:
Болезни: перевод описания для медицинского сканера

Ohar committed:
Merge pull request #1 from ss220-space/master220

Update from SS220

Ohar committed:
Merge branch 'master220' into vampire

# Conflicts:
#	code/game/gamemodes/vampire/vampire_powers.dm

Ohar committed:
Merge branch 'master220' into Poly_phrases

# Conflicts:
#	code/modules/mob/living/simple_animal/parrot.dm

Jordanstolethepreciousslam commented on code/datums/diseases/advance/symptoms/damage_converter.dm:
```diff
@@ -17,7 +17,7 @@ Bonus
 /datum/symptom/damage_converter
 
-	name = "Toxic Compensation"
+	name = "Компенсация токсинов"
```
Этот симптом не компенсирует токсины, наоборот, он компенсирует брут/берн токсина**ми**. Правильнее будет Токсическая компенсация или Компенсация токсинами.

Ohar replied:
Исправил

Jordanstolethepreciousslam commented on code/datums/diseases/advance/symptoms/choking.dm:
```diff
@@ -31,13 +31,13 @@ Bonus
 		switch(A.stage)
 			if(1, 2)
-				to_chat(M, "<span class='warning'>[pick("You're having difficulty breathing.", "Your breathing becomes heavy.")]</span>")
+				to_chat(M, "<span class='warning'>[pick("Вам становится трудно дышать.", "Ваше начинаете тяжело дышать.")]</span>")
```
Что

Ohar replied:
@Jordanstolethepreciousslam Что «что»?

Jordanstolethepreciousslam replied:
> Ваше начинаете тяжело дышать

Jordanstolethepreciousslam replied:
@Ohar

Ohar replied:
@Jordanstolethepreciousslam Поправил

Ohar replied:
Поправил

Jordanstolethepreciousslam commented on code/datums/diseases/anxiety.dm:
```diff
@@ -1,41 +1,41 @@
 		if(4)
 			if(prob(10))
-				to_chat(affected_mob, "<span class='danger'>You feel butterflies in your stomach.</span>")
+				to_chat(affected_mob, "<span class='danger'>Вы чувствуете, как у вас в животе порхают бабочек.</span>")
```
Кто порхает бабочек?

Ohar replied:
Исправил

Ohar committed:
Компенсация токсинов → Компенсация токсинов
https://github.com/ss220-space/Paradise/pull/685#discussion_r823958899

Ohar committed:
Исправление опечатки
https://github.com/ss220-space/Paradise/pull/685#discussion_r823988498

Ohar committed:
Merge branch 'master220' into Symptoms

# Conflicts:
#	code/datums/diseases/advance/symptoms/epinephrine.dm
#	code/modules/mob/living/simple_animal/parrot.dm

Ohar committed:
Merge branch 'Poly_phrases' into Symptoms

# Conflicts:
#	code/modules/mob/living/simple_animal/parrot.dm

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825253398

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825253687

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825253959

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825255016

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825255096

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825255151

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825254864

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825253636

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825254176

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825254806

Ohar committed:
Fix mistype

Ohar committed:
Merge branch 'master220' into Swedish-l10n

Ohar committed:
Merge branch 'master220' into Swedish-l10n

Ohar committed:
Мелкие правки шведского акцента

Ohar committed:
Merge remote-tracking branch 'origin/Swedish-l10n' into Swedish-l10n

Ohar committed:
УБраны большая часть «ь» и «й», они сильно забивают речь

Ohar committed:
Упрощение шведского акцента

Ohar committed:
Добавлена функция склонения числительных

Ohar committed:
scoreboard: исправлены склонения числительных

Ohar committed:
Склонение числительных: Теперь правильно обрабатываются числа больше сотни

Ohar committed:
Склонение числительных: Добавлено значение по-умолчанию

Ohar committed:
Update code/__HELPERS/pronouns.dm

Co-authored-by: Vallat <vox.crit2013@gmail.com>

Ohar committed:
Merge branch 'master220' into Round_statistics

Ohar committed:
Merge branch 'master220' into Swedish-l10n

Ohar committed:
Merge remote-tracking branch 'origin/Round_statistics' into Round_statistics

Ohar committed:
Функция склонения оптимизирована и упрощена

Ohar committed:
Merge remote-tracking branch 'ss220/master220' into vampire

Ohar committed:
Merge branch 'Swedish-l10n' into vampire

Ohar committed:
Merge branch 'Round_statistics' into vampire

Ohar committed:
Функция гендеризации «genderize_ru»

Ohar committed:
Вампир: перевод стадий сытости

Ohar committed:
Вампир: задачи набора крови

Ohar committed:
Вампир: перевод админских действий

Ohar committed:
Вампир: перевод сообщений чата и итогов раунда

Ohar committed:
declension_ru: исправлена обработка чисел, оканчивающихся на «4»

Ohar committed:
Merge branch 'Swedish-l10n' into Round_statistics

Ohar committed:
Правка была не нужна, код был правильный  😃
![изображение](https://user-images.githubusercontent.com/3854741/159117453-2b677ecc-9b28-4768-b077-52486d03e04f.png)

Ohar committed:
declension_ru: обработка исключений, проверка на целое число

Ohar committed:
declension_ru: нецелые числа и не числа → двойственное число

Ohar committed:
Update code/__HELPERS/pronouns.dm

Co-authored-by: Vallat <vox.crit2013@gmail.com>

Ohar committed:
Окно итогов раунда: мелкие оптимизации и правки

Ohar committed:
Merge remote-tracking branch 'origin/Round_statistics' into Round_statistics

Ohar committed:
Добыто руды: теперь тоже склоняется

Ohar committed:
Update code/game/gamemodes/scoreboard.dm

Co-authored-by: Aziz Chynaliev <azizonkg@gmail.com>

Ohar committed:
Слова теперь не разбиты на части

Ohar committed:
Merge remote-tracking branch 'origin/Round_statistics' into Round_statistics

Ohar committed:
Правка опечатки

Ohar committed:
Merge branch 'master220' into Symptoms

# Conflicts:
#	code/datums/diseases/advance/symptoms/epinephrine.dm
#	code/datums/diseases/advance/symptoms/painkiller.dm
#	code/modules/mob/living/simple_animal/parrot.dm

Ohar committed:
Merge branch 'master220' into vampire

Ohar committed:
Merge branch 'Round_statistics' into vampire

Ohar committed:
Merge branch 'master220' into vampire

Ohar committed:
Fix code style

Ohar committed:
Merge branch 'master220' into Symptoms

Ohar committed:
Merge branch 'vampire' into Symptoms

Ohar committed:
Перевод болезней

Ohar committed:
Перевод органов похитителей

Ohar committed:
Болезнетворные агенты

Ohar committed:
/datum/species: fix missed props

Ohar committed:
Правка названий путей распространения болезней

Ohar committed:
/datum/disease/tuberculosis
Правка названия

Ohar committed:
Merge branch 'Symptoms' into Reagents

Ohar committed:
/datum/disease/tuberculosis
Правка названия путей распространения

Ohar committed:
/datum/disease/brainrot
Правка названия

Ohar committed:
/datum/disease/brainrot
перевод симптома

Ohar committed:
/datum/disease/brainrot
fix code style

girafe969 commented on code/datums/diseases/_disease.dm:
```diff
@@ -39,10 +39,10 @@ GLOBAL_LIST_INIT(diseases, subtypesof(/datum/disease))
+	var/name = "Без заболеваний"
 	var/desc = ""
-	var/agent = "some microbes"
+	var/agent = "некоторые микробы"
```
В русском так не говорится, в данном случае дословный перевод будет "какие-то микробы" или "сколько-то микробов". Можно оставить просто "микробы"

Ohar replied:
```suggestion
	var/agent = "какие-то микробы"
```

girafe969 commented on code/datums/diseases/advance/advance.dm:
```diff
@@ -24,12 +24,12 @@ GLOBAL_LIST_INIT(advance_cures, list(
+	name = "Без названия" // We will always let our Virologist name our disease.
 	desc = "Спроектированная болезнь, может содержать сразу несколько симптомов."
 	form = "Улучшенная болезнь" // Will let med-scanners know that this disease was engineered.
 	agent = "advance microbes"
```
Улучшенные/продвинутые микробы

Ohar replied:
Резонно

Ohar replied:
```suggestion
	agent = "улучшенные микробы"
```

girafe969 commented on code/datums/diseases/advance/symptoms/beard.dm:
```diff
@@ -33,17 +33,17 @@ BONUS
 						H.update_fhair()
 				if(3, 4)
-					to_chat(H, "<span class='warning'>You feel tough.</span>")
+					to_chat(H, "<span class='warning'>Вы чувствуете себя на 20% круче.</span>")
```
А почему только на 20%?)

Ohar replied:
Потому что «20% cooler» это мем, который тут очень уместен)

girafe969 commented on code/datums/diseases/advance/symptoms/cough.dm:
```diff
@@ -31,7 +31,7 @@ BONUS
 		switch(A.stage)
 			if(1, 2, 3)
-				to_chat(M, "<span notice='warning'>[pick("You swallow excess mucus.", "You lightly cough.")]</span>")
+				to_chat(M, "<span notice='warning'>[pick("Вы сглатываете лишнюю мокроту.", "Вы покашливаете.")]</span>")
```
Мокрота и так лишняя. Не лишняя слизь и секреции слизистых (mucus), которые, при излишнем образовании и отделении, называются мокротой (как я понял из 5 минут общения с гуглом)

Ohar replied:
Если убрать «лишнюю», смысл немного изменится - не будет очевидно что происходит что-то ненормальное

girafe969 replied:
Можно использовать "подступившую". Так будет понятно, что её не было

girafe969 commented on code/datums/diseases/advance/symptoms/deafness.dm:
```diff
@@ -31,13 +31,13 @@ Bonus
 			if(5)
 				if(!(DEAF in M.mutations))
-					to_chat(M, "<span class='userdanger'>Your ears pop and begin ringing loudly!</span>")
+					to_chat(M, "<span class='userdanger'>Вы слышите потрескивание и нарастающий гул!</span>")
```
Насколько я знаю, ear pop это когда ухо закладывает, впервые слышу про треск.

Ohar replied:
А ringing?
Но да, стоит переформулировать

Ohar replied:
```suggestion
					to_chat(M, "<span class='userdanger'>Вы слышите нарастающие гул и треск, от которых закладывает уши!</span>")
```

girafe969 replied:
Ringing это звон в ушах

girafe969 commented on code/datums/diseases/advance/symptoms/fire.dm:
```diff
@@ -31,16 +31,16 @@ Bonus
 		switch(A.stage)
 			if(3)
-				to_chat(M, "<span class='warning'>[pick("You feel hot.", "You hear a crackling noise.", "You smell smoke.")]</span>")
+				to_chat(M, "<span class='warning'>[pick("Вы чувствуете жар.", "У вас жар.", "Вы слышите потрескивающие звуки.", "Вы чувствуете запах дыма.")]</span>")
```
В русском мы всё же чаще чувствуем запах гари, чем запах дыма.

Ohar replied:
Верно

Ohar replied:
```suggestion
				to_chat(M, "<span class='warning'>[pick("Вы чувствуете жар.", "У вас жар.", "Вы слышите потрескивающие звуки.", "Вы чувствуете запах гари.")]</span>")
```

girafe969 commented on code/datums/diseases/advance/symptoms/flesh_eating.dm:
```diff
@@ -31,9 +31,9 @@ Bonus
 		switch(A.stage)
 			if(2,3)
-				to_chat(M, "<span class='warning'>[pick("You feel a sudden pain across your body.", "Drops of blood appear suddenly on your skin.")]</span>")
+				to_chat(M, "<span class='warning'>[pick("Вы чувствуете резкую боль во всём теле.", "На вашей кожей внезапно проступают капельки крови.")]</span>")
```
На вашей коже

Ohar replied:
Коже?

Ohar replied:
Поправлю

girafe969 commented on code/datums/diseases/advance/symptoms/flesh_eating.dm:
```diff
@@ -31,9 +31,9 @@ Bonus
+				to_chat(M, "<span class='warning'>[pick("Вы чувствуете резкую боль во всём теле.", "На вашей кожей внезапно проступают капельки крови.")]</span>")
 			if(4,5)
-				to_chat(M, "<span class='userdanger'>[pick("You cringe as a violent pain takes over your body.", "It feels like your body is eating itself inside out.", "IT HURTS.")]</span>")
+				to_chat(M, "<span class='userdanger'>[pick("Вы содрогаетесь от жуткой боли во всём теле.", "Ваше тело, кажется, пожирает само себя прямо внутри вас!", "ВАМ ОЧЕНЬ БОЛЬНО!")]</span>")
```
Ещё бы тело пожирало само себя снаружи вас, вообще фильм ужасов бы получился. В русском схожее значение имеет "Ваше тело будто выворачивает наизнанку", либо просто убрать "прямо внутри вас!"

Ohar replied:
Так там и есть фильм ужасов по факту

girafe969 replied:
Либо заменить прямо внутри вас на изнутри

girafe969 commented on code/datums/diseases/advance/symptoms/hallucigen.dm:
```diff
@@ -31,11 +31,11 @@ Bonus
 		switch(A.stage)
 			if(1, 2)
-				to_chat(M, "<span class='warning'>[pick("Something appears in your peripheral vision, then winks out.", "You hear a faint whispher with no source.", "Your head aches.")]</span>")
+				to_chat(M, "<span class='warning'>[pick("Что-то мелькает на границе вашего периферийного зрения, и сразу исчезает.", "Вы слышите лёгкий шёпот из неопределённого источника.", "У вас болит голова.")]</span>")
```
Периферийное зрение и так на границе зрения, это тавтология. Лучше либо переделать, оставив периферийное зрение и убрав границу, либо просто убрать периферийного. 
"шепот из ниоткуда" не подойдёт? Как-то "неопределенного источника" выглядит громоздко

Ohar replied:
Ок, поменяю

Ohar replied:
```suggestion
				to_chat(M, "<span class='warning'>[pick("Что-то мелькает на границе вашего зрения, и сразу исчезает.", "Вы слышите лёгкий шёпот непонятно откуда.", "У вас болит голова.")]</span>")
```

girafe969 commented on code/datums/diseases/advance/symptoms/headache.dm:
```diff
@@ -30,5 +30,5 @@ BONUS
 	if(prob(SYMPTOM_ACTIVATION_PROB))
 		var/mob/living/M = A.affected_mob
-		to_chat(M, "<span class='warning'>[pick("Your head hurts.", "Your head starts pounding.")]</span>")
+		to_chat(M, "<span class='warning'>[pick("У вас болит голова.", "Голова наливается тяжестью.")]</span>")
```
Голова или раскалывается, или кружится, или башка трещит, или виски стучат. Наливается тяжестью редко используют для описания мигрени

girafe969 commented on code/datums/diseases/advance/symptoms/oxygen.dm:
```diff
@@ -17,7 +17,7 @@ Bonus
 /datum/symptom/oxygen
 
-	name = "Self-Respiration"
+	name = "Автодыхание"
```
Не очень нравится это название, так как дыхание и так происходит автоматически на уровне рефлексов. Пока придумать альтернативу не могу, Самодыхание с Вики тоже звучит убого. Респирация это всё же немного не то же самое, что дыхание. Ещё подумаю, может что придет в голову

girafe969 commented on code/datums/diseases/anxiety.dm:
```diff
@@ -1,41 +1,41 @@
+	spread_text = "Контактный"
 	spread_flags = CONTACT_GENERAL
-	cure_text = "Ethanol"
+	cure_text = "Этанол"
```
Не стоит ли тут и в последующих местах добавить в скобках английское название лекарства? Или раз уж переводим, то всё?

girafe969 commented on code/datums/diseases/appendicitis.dm:
```diff
@@ -1,13 +1,13 @@
 	permeability_mod = 1
-	desc = "If left untreated the subject will become very weak, and may vomit often."
-	severity = "Dangerous!"
+	desc = "Если не вылечить, то субъект будет слабеть и может начать постоянно блевать."
```
Блевать - разговорное слово. Использовать его в научном описании болезни неуместно. (Учитывая, что у тебя в том же предложении есть научное слово субъект)

Ohar replied:
А что уместно?

girafe969 replied:
Субъект будет слабеть и часто испытывать рвотные позывы

girafe969 commented on code/datums/diseases/brainrot.dm:
```diff
@@ -1,18 +1,18 @@
-	cure_chance = 15//higher chance to cure, since two reagents are required
-	desc = "This disease destroys the braincells, causing brain fever, brain necrosis and general intoxication."
+	cure_chance = 15 // higher chance to cure, since two reagents are required
+	desc = "Эта болезнь разрушает клетки мозга, провоцируя воспаление мозга, его некроз и общую интоксикацию организма."
```
Убери тут про общую интоксикацию, это давно выпилили, но не поменяли описание. (Буквально в комменте ниже написано)

Ohar replied:
```suggestion
	desc = "Эта болезнь разрушает клетки мозга, провоцируя его воспаление и последующий некроз."
```

girafe969 commented on code/datums/diseases/cold.dm:
```diff
@@ -1,13 +1,13 @@
 	max_stages = 3
 	spread_flags = AIRBORNE
-	cure_text = "Rest & Spaceacillin"
+	cure_text = "Отдых + Spaceacillin"
```
А спейсацелин особенный? (Хотя вообще я бы перевел его как космоцелин, но на Вики так)

girafe969 reviewed (changes requested):
Короче меня достало, остановился на простуде9 , потом посмотрю чё там дальше. Не забудь обновить Вики под твои названия

girafe969 commented on code/datums/diseases/beesease.dm:
```diff
@@ -1,38 +1,38 @@
+	cure_text = "Сахар"
 	cures = list("sugar")
-	agent = "Apidae Infection"
+	agent = "Пчелиный вирус"
```
Может можно тут как-то сохранить научный окрас текста? Я понимаю, что на русском это семейство пчелиных, и у нас нет "завумного" названия для них, но хотелось бы как-то это обыграть

girafe969 commented on code/datums/diseases/critical.dm:
```diff
@@ -19,9 +19,9 @@
-	spread_text = "The patient is in shock"
+	name = "Шок"
+	form = "Требуется неотложная помощь"
+	spread_text = "Пациент в шоке"
```
Звезда в шоке!))
Пациент в состоянии шока/ в шоковом состоянии

girafe969 commented on code/datums/diseases/critical.dm:
```diff
@@ -37,55 +37,55 @@
 					affected_mob.emote(pick("shiver", "pale", "moan", "shudder", "tremble"))
 				if(prob(5))
-					to_chat(affected_mob, "<span class='danger'>You feel absolutely terrible!</span>")
+					to_chat(affected_mob, "<span class='danger'>Вам становится катастрофически плохо!</span>")
```
Не становится, тут нет сравнения с тем, как было раньше

girafe969 commented on code/datums/diseases/critical.dm:
```diff
@@ -37,55 +37,55 @@
-	spread_text = "The patient is having a cardiac emergency"
+	name = "Сердечная недостаточность"
+	form = "Требуется неотложная помощь"
+	spread_text = "У пациента сердечная недостаточность"
```
Сердечный приступ

girafe969 commented on code/datums/diseases/critical.dm:
```diff
@@ -108,27 +108,27 @@
 					affected_mob.emote(pick("pale", "groan"))
 				if(prob(5))
-					to_chat(affected_mob, "<span class='danger'>Your heart lurches in your chest!</span>")
+					to_chat(affected_mob, "<span class='danger'>Ваше сердце колотится в груди!</span>")
```
Оно и так колотится в груди. Стоит добавить что-то вроде "бешено колотится", чтобы показать что это не нормально

Ohar committed:
Update code/datums/diseases/advance/symptoms/hallucigen.dm

Co-authored-by: girafe969 <88498986+girafe969@users.noreply.github.com>

girafe969 commented on code/datums/diseases/critical.dm:
```diff
@@ -197,8 +197,8 @@
 					affected_mob.Weaken(6)
 					affected_mob.Stuttering(10)
-					to_chat(affected_mob, "<span class='warning'>You feel [pick("numb", "confused", "dizzy", "lightheaded")].</span>")
+					to_chat(affected_mob, "<span class='warning'>Вы чувствуете [pick("онемение", "замешательство", "головокружение", "lightheaded")].</span>")
```
Lightheaded на русском это что-то вроде "потемнение в глазах". Предобморочное состояние короче. Так как на английском lightheaded и dizzy - синонимы, чтобы не делать тут новую конструкцию lightheaded можно вообще убрать

Ohar committed:
Update code/datums/diseases/advance/advance.dm

Поправил микробов

Co-authored-by: girafe969 <88498986+girafe969@users.noreply.github.com>

Ohar committed:
Update code/datums/diseases/_disease.dm

Поправил

Co-authored-by: girafe969 <88498986+girafe969@users.noreply.github.com>

Ohar committed:
Update code/datums/diseases/advance/symptoms/deafness.dm

Co-authored-by: girafe969 <88498986+girafe969@users.noreply.github.com>

Ohar committed:
Update code/datums/diseases/advance/symptoms/fire.dm

Co-authored-by: girafe969 <88498986+girafe969@users.noreply.github.com>

Ohar committed:
Update code/datums/diseases/brainrot.dm

Раз интоксикация убрана, то и описание стоит сразу подправить
https://github.com/ss220-space/Paradise/pull/685#discussion_r851605376

Co-authored-by: girafe969 <88498986+girafe969@users.noreply.github.com>

girafe969 commented on code/datums/diseases/dna_spread.dm:
```diff
@@ -1,16 +1,17 @@
 	var/transformed = 0
 	desc = "This disease transplants the genetic code of the initial vector into new hosts."
+	desc = "Это заболевание переносит генетический код начального вектора в новых носителей."
 	severity = MEDIUM
```
Не убрал английское описание. Я бы перевел initial vector как "нулевого пациента" или "вектора инициализации" (если это отсылка на него).
Как я понял спустя полчаса общения с гуглом, скорее всего тут игра слов между initial patient (первый заразившийся каким-либо вирусом) и initialization vector (криптографическое понятие), как это правильно обыграть на русском я не уверен.

girafe969 commented on code/datums/diseases/fake_gbs.dm:
```diff
@@ -1,13 +1,13 @@
 	cure_text = "Diphenhydramine & Sulfur"
 	cures = list("diphenhydramine","sulfur")
-	agent = "Gravitokinetic Bipotential SADS-"
+	agent = "Гравитокинетический бипотенциал SADS−"
```
"бипотенциальный СВС-" (синдром внезапной смерти)

girafe969 commented on code/datums/diseases/gbs.dm:
```diff
@@ -1,12 +1,12 @@
 	cures = list("diphenhydramine","sulfur")
 	cure_chance = 15//higher chance to cure, since two reagents are required
-	agent = "Gravitokinetic Bipotential SADS+"
+	agent = "Гравитокинетический бипотенциал SADS+"
```
бипотенциальный СВС+

girafe969 commented on code/datums/diseases/kingstons.dm:
```diff
@@ -17,57 +17,57 @@
-					affected_mob.visible_message("<span class='danger'>[affected_mob]'s form contorts into something more feline!</span>", \
-													"<span class='userdanger'>YOU TURN INTO A TAJARAN!</span>")
+					affected_mob.visible_message("<span class='danger'>Черты [affected_mob] становятся намного более кошачьими!</span>", \
+													"<span class='userdanger'>ВЫ ПРЕВРАТИЛИСЬ В ТАЯРАНА!</span>")
```
В таяру. Тайаран это название расы, таяра - представитель этой расы

girafe969 commented on code/datums/diseases/kingstons.dm:
```diff
@@ -17,57 +17,57 @@
 	chosensuff = pick(virsuffix)
 
-	name = "[initial(chosentype.name)] [chosensuff]"
+	name = "[initial(chosentype.name_adjective_female)] [chosensuff]"
```
Я так понял, тут должно быть типа "Скреллская чума" на выходе? Чувствую, будет пиздец. Наверное, стоит пока что это не трогать.

girafe969 commented on code/datums/diseases/kingstons.dm:
```diff
@@ -76,22 +76,22 @@
-						twisted.visible_message("<span class='danger'>[twisted]'s skin splits and form contorts!</span>", \
-														"<span class='userdanger'>Your body mutates into a [initial(chosentype.name)]!</span>")
+						twisted.visible_message("<span class='danger'>Кожа [twisted] рвётся и вытягивается!</span>", \
+														"<span class='userdanger'>Ваше тело мутирует в [initial(chosentype.name)]!</span>")
```
Тут тоже будет склонение, и на строке выше

girafe969 commented on code/datums/diseases/kingstons.dm:
```diff
@@ -17,57 +17,57 @@
 				else
-					affected_mob.visible_message("<span class='danger'>[affected_mob]'s form contorts into something more feline!</span>", \
-													"<span class='userdanger'>YOU TURN INTO A TAJARAN!</span>")
+					affected_mob.visible_message("<span class='danger'>Черты [affected_mob] становятся намного более кошачьими!</span>", \
```
Привет склонение имён по падежам, удачи тебе с этим)

girafe969 commented on code/datums/diseases/lycancoughy.dm:
```diff
@@ -17,37 +17,37 @@
 				var/mob/living/puppypicked = pick(puppytype)
-				affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up [initial(puppypicked.name)]!</span>", \
-													"<span class='userdanger'>You cough up [initial(puppypicked.name)]?!</span>")
+				affected_mob.visible_message("<span class='danger'>[affected_mob] выкашливает [initial(puppypicked.name)]!</span>", \
```
"Павел Михайлов выкашливает fox".
Ниже всё так же

girafe969 commented on code/datums/diseases/magnitis.dm:
```diff
@@ -1,22 +1,22 @@
 	cure_text = "Iron"
 	cures = list("iron")
-	agent = "Fukkos Miracos"
+	agent = "Фуккос Миракос"
```
Это сто процентов игра слов. Miracos скорее всего от miraculous. Fukkos возможо от Fukko ( https://www.google.com/amp/s/www.urbandictionary.com/define.php%3fterm=Fukko&amp=true), хотя я не уверен. Дословно что-то типа Шлюхус Чудеснус/Необыкновеннус. Только это больше подражание латинскому, а в оригинале - испанскому

girafe969 commented on code/datums/diseases/magnitis.dm:
```diff
@@ -44,9 +44,9 @@
+				to_chat(affected_mob, "<span class='danger'>По вашему телу проходят мощные колебания.</span>")
 			if(prob(2))
-				to_chat(affected_mob, "<span class='danger'>You query upon the nature of miracles.</span>")
+				to_chat(affected_mob, "<span class='danger'>Вы задаётесь вопросом о природе происходящих странностей.</span>")
```
Опять же отсылка на Miracos в агенте. "о природе чудес" (хотя звучит не очень)
Суть не в происходящих сейчас странностях (из-за вируса), а о странностях в целом. Если сможешь это по-другому сформулировать, будет круто

girafe969 commented on code/datums/diseases/retrovirus.dm:
```diff
@@ -18,7 +18,7 @@
 /datum/disease/dna_retrovirus/New()
 	..()
-	agent = "Virus class [pick("A","B","C","D","E","F")][pick("A","B","C","D","E","F")]-[rand(50,300)]"
+	agent = "Вирус класса [pick("А","Б","В","Г","Д","Е")][pick("А","Б","В","Г","Д","Е")]-[rand(50,300)]"
```
Массив букв переводить не стоит, так как это отсылка на двоичную систему

girafe969 commented on code/datums/diseases/transformation.dm:
```diff
@@ -1,20 +1,20 @@
-	cure_text = "A coder's love (theoretical)."
-	agent = "Shenanigans"
+	cure_text = "Любовь кодера (теоретически)."
+	agent = "Шалости"
```
"Шалости" недостаточно сленгово, как по мне. Я бы сделал "проделки педалей"

girafe969 commented on code/datums/diseases/transformation.dm:
```diff
@@ -106,31 +106,30 @@
 		if(2)
 			if(prob(2))
-				to_chat(affected_mob, "<span class='notice'>Your [pick("back", "arm", "leg", "elbow", "head")] itches.</span>")
+				to_chat(affected_mob, "<span class='notice'>Ваша [pick("спина", "рука", "нога", "грудь", "голова")] чешется.</span>")
```
Скопируй лист из чесотки, эта болезнь пытается под неё маскироваться на ранних стадиях.
Ты вроде там ещё задницу добавлял и что-то ещё

girafe969 commented on code/datums/diseases/transformation.dm:
```diff
@@ -106,31 +106,30 @@
-	stage3	= list("<span class='danger'>Your joints feel very stiff.</span>", "Your skin feels loose.", "<span class='danger'>You can feel something move...inside.</span>")
-	stage4	= list("<span class='danger'>Your skin feels very loose.</span>", "<span class='danger'>You can feel... something...inside you.</span>")
-	stage5	= list("<span class='danger'>Your skin feels as if it's about to burst off!</span>")
+	stage2	= list("Ваши суставы теряют гибкость.", "<span class='danger'>Биg… буп…</span>")
```
Бип... Буп...

girafe969 commented on code/datums/diseases/transformation.dm:
```diff
@@ -139,58 +138,57 @@
-	stage5	= list("<span class='danger'>Your skin feels as if it's about to burst off!</span>")
+	stage2	= list("У вас чешется горло.", "<span class='danger'>Убить…</span>")
+	stage3	= list("<span class='danger'>У вас очень сильно чешется горло.</span>", "Ваша кожа твердеет.", "<span class='danger'>Вы чувствуете, как что-то двигается внутри….</span>")
+	stage4	= list("<span class='danger'>Ваша кожа значительно твердеет.</span>", "<span class='danger'>Ваша кровь начинает кипеть!</span>", "<span class='danger'>Вы чувствуете… что-то… внутри себя.</span>")
```
Кровь закипает

girafe969 commented on code/datums/diseases/transformation.dm:
```diff
@@ -139,58 +138,57 @@
-	stage4	= list("<span class='danger'>You are turning into a slime.</span>")
-	stage5	= list("<span class='danger'>You have become a slime.</span>")
+	stage1	= list("Вам не очень хорошо.")
+	stage2	= list("Ваша кожа становится липкой.")
```
Слизкой

girafe969 commented on code/datums/diseases/tuberculosis.dm:
```diff
@@ -1,14 +1,14 @@
 	viable_mobtypes = list(/mob/living/carbon/human)
 	cure_chance = 5//like hell are you getting out of hell
-	desc = "A rare highly transmittable virulent virus. Few samples exist, rumoured to be carefully grown and cultured by clandestine bio-weapon specialists. Causes fever, blood vomiting, lung damage, weight loss, and fatigue."
+	desc = "Редкий высококонтагиозный вирулентный вирус. Существует несколько штаммов, которые, по слухам, были тщательно выращены и культивированы нелегальными специалистами по биологическому оружию. Вызывает лихорадку, кровавую рвоту, повреждение лёгких, истощение и слабость."
```
По слову истощение непонятно, что ты будешь более голодным. Лучше перевести дословно - потерю веса

girafe969 commented on code/datums/diseases/tuberculosis.dm:
```diff
@@ -19,41 +19,41 @@
 		if(5)
 			if(prob(2))
-				to_chat(affected_mob, "<span class='userdanger'>[pick("You feel your heart slowing...", "You relax and slow your heartbeat.")]</span>")
+				to_chat(affected_mob, "<span class='userdanger'>[pick("Ваше сердце начинает биться реже…", "Вы расслабляетесь и сердцебиение замедляется.")]</span>")
```
Тут не надо запятую? Я уже не в состоянии гуглить

girafe969 commented on code/datums/diseases/tuberculosis.dm:
```diff
@@ -19,41 +19,41 @@
 				affected_mob.adjust_nutrition(-100)
 			if(prob(15))
-				to_chat(affected_mob, "<span class='danger'>[pick("You feel uncomfortably hot...", "You feel like unzipping your jumpsuit", "You feel like taking off some clothes...")]</span>")
+				to_chat(affected_mob, "<span class='danger'>[pick("Вв чувствуете неприятный жар…", "Вам хочется расстегнуть комбинезон.", "Вам хочется снять одежду…")]</span>")
```
Вы, а не Вв

girafe969 commented on code/datums/diseases/wizarditis.dm:
```diff
@@ -1,15 +1,15 @@
 	disease_flags = CAN_CARRY|CAN_RESIST|CURABLE
 	permeability_mod = 0.75
-	desc = "Some speculate, that this virus is the cause of Wizard Federation existance. Subjects affected show the signs of dementia, yelling obscure sentences or total gibberish. On late stages subjects sometime express the feelings of inner power, and, cite, 'the ability to control the forces of cosmos themselves!' A gulp of strong, manly spirits usually reverts them to normal, humanlike, conditi ...
+	desc = "Поговаривают, что этот вирус — причина существования Федерации Космических Волшебников. Поражённые им субъекты выказывают признаки деменции, выкрикивают невразумительные предложения и несут околесицу. На поздних стадиях субъекты иногда ощущают внутреннюю силу и, цитата, <i>«способность управлять силами самого космоса!»</i>. Глоток крепкого мужественного спирта обычно возвращает их в норм ...
```
Спирт не бывает мужественным. И это вообще не спирт. Глоток крепкого мужского спиртного

girafe969 commented on code/datums/diseases/wizarditis.dm:
```diff
@@ -1,15 +1,15 @@
 	cures = list("manlydorf")
 	cure_chance = 100
-	agent = "Rincewindus Vulgaris"
+	agent = "Ринсвиндий обыкновенный"
```
Тоже неплохо бы было адаптировать. Rincé - это смывать, полоскать на французском, то же что rinse на англе. Вторая часть либо от wind на английском и французском (как wind of magic), либо window на английском. Но мойщик окон на англе window washer, а не window rincer, поэтому всё же скорее всего это ветер. Только связать эти два слова в одно у меня не получается.

girafe969 reviewed (changes requested):
Бобик сдох... Я вроде закончил, там дальше всякие левые файлы
